_ml-commons-plugin/cluster-settings.md

```diff
@@ -99,34 +92,63 @@ Controls how many upload model tasks can run in parallel on one node. If set to
 
 ## Load model tasks per node
 
+Controls how many load model tasks can run in parallel on one node. If set as 0, you cannot load models to any node.
+
 ### Setting
 
 ```
 plugins.ml_commons.max_load_model_tasks_per_node: 10
 ```
 
-### Description
-
-Controls how many load model tasks can run in parallel on one node. If set as 0, you cannot load models to any node.
-
 ### Values 
 
 - Default value: `10`
 - Value range: [0, 10]
 
 ## Add trusted URL
 
+The default value allows uploading a model file from any http/https/ftp/local file. You can change this value to restrict trusted model URL.
```

Comment:
restrict trusted model URLs.  (not sure if it's supposed to be singular or plural)